src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+abstract class Figure implements Drawable, GetArea {
```

Comment:
You should not use abstract classes to set behavior for classes. In this case, it seems like you are using an abstract class to define common behavior for figures, which might be better suited for an interface or a regular class depending on how it's being used. Review whether you need an abstract class or if an interface or a regular class can fulfill your requirements.